ragnarok_formats/src/model.rs

```diff
@@ -115,15 +168,28 @@ pub struct NodeData {
     pub face_count: u32,
     #[repeating(face_count)]
     pub faces: Vec<FaceData>,
-    #[version_equals_or_above(2, 5)] // unsure what vesion this is supposed to be (must be > 1.5)
+    // TODO: Scale Keyframe not implemented yet
+    #[version_equals_or_above(1, 6)]
     #[new_derive]
-    pub position_keyframe_count: Option<u32>,
-    #[repeating_option(position_keyframe_count)]
-    pub position_keyframes: Vec<PositionKeyframeData>,
+    pub scale_keyframe_count: Option<u32>,
+    #[repeating_option(scale_keyframe_count)]
+    pub scale_keyframes: Vec<ScaleKeyframeData>,
     #[new_derive]
     pub rotation_keyframe_count: u32,
     #[repeating(rotation_keyframe_count)]
     pub rotation_keyframes: Vec<RotationKeyframeData>,
+    // TODO: Translation Keyframe not implemented yet
```

Comment:
What do you mean by not yet implemented? The parsing or using the data?

Reply:
The manipulation of the model

Reply:
The we should either remove the comments or put it somewhere else. Something like `TODO: Use translations keyframes` in the `load` method of the `ModelLoader` is less likely to become a stale comment.